templates/CRM/Contribute/Form/Selector.tpl

```diff
@@ -65,8 +65,9 @@
     <td>{$row.contact_type}</td>
       <td><a href="{crmURL p='civicrm/contact/view' q="reset=1&cid=`$row.contact_id`"}">{$row.sort_name}</a></td>
     {/if}
-    {assign var="targetRowID" value="paymentDetails"|cat:$row.contribution_id}
-    {include file='CRM/common/expandRow.tpl' rowEntityID=$row.contribution_id rowEntity='contribution' targetRowID=$targetRowID baseUrl='payment'}
+    <td>
+      <a class="crm-expand-row" href="{crmURL p='civicrm/payment' q="view=transaction&component=contribution&action=browse&cid=`$row.contact_id`&id=`$row.contribution_id`&selector=1"}"></a>
+    </td>
```

Comment:
I didn't start from scratch. I started from what was happening on the pledge tab & took that as my starting point. I don't have a problem with the change you suggest - other than that is should potentially be consistent with pledge tab.

NB just to mention it so at least I've said it - I feel like memberships & events would ideally expand to show contributions and/or membership log lines but in my thinking not line items. Although they are linked to memberships & events in the table I feel we should encourage people to conceptualise  line items as sub entities on contributions